manifests/params.pp

```diff
@@ -16,30 +16,55 @@
       $systemd            = false # by default
     } # end osfamily Gentoo
     'Debian': {
-      # Debian 7 uses version 6.8
-      # Debian 8 uses version 6.8
-      # Debian 9 uses version 6.8
-      # Ubuntu 12.04 uses version 6.7
-      # Ubuntu 14.04 uses version 6.8
-      # Ubuntu 16.04 uses version 6.8
+      # Debian 7 uses version 0.6.8
+      # Debian 8 uses version 0.6.8
+      # Debian 9 uses version 0.6.9
+      # Debian 10 uses version 0.6.9
+      # Debian 11 uses version 0.6.11
+      # Ubuntu 12.04 uses version 0.6.7
+      # Ubuntu 14.04 uses version 0.6.8
+      # Ubuntu 16.04 uses version 0.6.8
+      # Ubuntu 18.04 uses version 0.6.9
+      # Ubuntu 20.04 uses version 0.6.11
```

Comment:
Can we add Ubuntu 22.04 as well? The module path there is also `/usr/lib/${gcc_arch}-linux-gnu/pkcs11/opensc-pkcs11.so` and the version is `0.6.11`.

Reply:
Hmm... Do you know if puppet is even supported on 22.04? AFAIK ubuntu 22.04 switched to a newer openssl release, which is not supported by ruby <= 2.7 and puppet is not yet ported over to ruby>=3...

For instance I cannot find any puppet packages for ubuntu 22.04 here: http://apt.puppetlabs.com/ so I cannot test this with test-kitchen...

I also cannot integrate ubuntu 22.04 into the gitlab-actions test, since they do not support ruby 2.7 there: https://github.com/ruby/setup-ruby#supported-platforms

Reply:
I seems I was mistaken. Apparently ruby 3 is supported by puppet (at least the ['support ruby 3' issue](https://tickets.puppetlabs.com/browse/PUP-11071) is closed). However it seems they still do not provide puppet packages for ubuntu 22.04. And dependent puppet modules like `puppet-module-postfix` also doesn't appear to have ruby 3 support yet: https://rubygems.org/search?query=puppet-module-posix

Reply:
Indeed, the Puppet support for Ubuntu 22.04 is not there yet. Considering that this makes testing difficult/impossible, let's table this for now. I will create a follow-up issue for this.